package/upgrade/upgrade_manifests.sh

```diff
@@ -405,6 +415,21 @@ EOF
   wait_kubevirt harvester-system kubevirt $REPO_KUBEVIRT_VERSION
 }
 
+check_has_harvester_longhorn_sc() {
```

Comment:
Are the following two functions not used?

Reply:
The next commit will remove.